osbs/build/build_requestv2.py

```diff
@@ -144,6 +146,14 @@ def render_resource_limits(self):
             resources['limits'] = limits
             self.template['spec']['resources'] = resources
 
+    def render_resource_requests(self):
+        if self._resource_requests is not None:
+            resources = self.template['spec'].setdefault('resources', {})
+            requests = resources.setdefault('requests', {})
+            requests.update(self._resource_requests)
+            resources['requests'] = requests
+            self.template['spec']['resources'] = resources
```

Comment:
with `setdefault` you can just remove these 2 lines. That was my point.

Reply:
ok